hopr-socks/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "hopr-socks"
+version = "0.1.0"
+authors = ["HOPR Association <tech@hoprnet.org>"]
+edition = "2021"
+description = ""
+homepage = "https://hoprnet.org/"
+repository = "https://github.com/hoprnet/hoprnet"
+license = "GPL-3.0-only"
+
+[dependencies]
+fast-socks5 = "0.9.6"
+env_logger = "0.10.2"
+structopt = "0.3"
+tokio = { version = "1", features = [
```

Comment:
Would be good to use `workspace = true` instead of explicit `version`.

That also applies to `env_logger` or rather use `tracing` crate right away instead.

Reply:
Addressed.

Reply:
Yes, pls, tracing.